reproduce-figures/README.md

```diff
@@ -1,10 +1,75 @@
 This directory contains a script and certain data files necessary for reproducing manuscript figures, tables, and analyses.
+It contains the following contents:
 
-The script `prepare-scpca-portal-data.R` organizes and prepares data obtained from the ScPCA Portal and S3 needed to reproduce figures and analyses.
-**CAUTION: Running this script requires at least 170 GB of free disk space.**
+* The script `prepare-scpca-portal-data.R` prepares data obtained from the ScPCA Portal and AWS S3 needed to reproduce figures, tables, and the bulk RNA-Seq analysis.
+Please see instructions below for usage.
   * This script uses functions provided in `utils.R`
-  * This script also uses the `scratch` directory during file processing.
+  * This script also uses the `./scratch` directory during file processing
+* TSV files `FigS1B-gene-expression-data.tsv` and `FigS1C_FigS1D-cell-metrics-data.tsv` are used as input to the script `../scripts/FigS1B-D_method-metrics-comparison.R` which creates Figures S1-D.
```

Comment:
```suggestion
* TSV files, `FigS1B-gene-expression-data.tsv` and `FigS1C_FigS1D-cell-metrics-data.tsv`, which are used as input to the script `../scripts/FigS1B-D_method-metrics-comparison.R` to create Figures S1-D.
```
I think this just clarifies that these files live here but are used as input to another script.